k4e-agent-sti.sh

```diff
@@ -66,6 +66,13 @@ if ! subscription-manager refresh; then
    exit 1
 fi
 
+#---------------------------------
+# Cleanup before running
+#---------------------------------
+rm -rf ~/rpmbuild/*
+rm -rf /home/builder
+rm -rf /var/www/html/k4e-repo
+
```

Comment:
let's delete also `/var/www/html/${IMAGE_NAME}` in case the same command will be tried more than once with the same image name

Reply:
I don't think it's good idea,
If something will went wrong you will not have any image at all...